tests/macsec/__init__.py

```diff
@@ -115,6 +120,8 @@ def filter(interface, neighbor, mg_facts, tbinfo):
         if neighbor["name"] not in nbrhosts.keys():
             return
         port = mg_facts["minigraph_neighbors"][interface]["port"]
+        if tbinfo["topo"]["type"] == "t2" and is_interface_portchannel_member(mg_facts['minigraph_portchannels'], interface):
```

Comment:
Why is condition `is_interface_portchannel_member` only for topology t2? Any limitation? Please add comments to describe the reason.

Reply:
Yes, currently macsec on portchannel don't work on J2C+ based cards, will remove this check once the support is added.

Reply:
updated